index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
+        <nav class="nav-bar">
+            <a href="#" class="logo-name">Seth Bizimana</a>
+           <a href="#" class="menu-icon">
+                <img alt="icon of menu button" src="images/Union.png">
+           </a> 
```

Comment:
- [ ] As mentioned by the previous reviewer your design is not matching with Figma design kindly stick with Figma design. please Use the `font-family: 'Roboto';`
### your design
![his](https://user-images.githubusercontent.com/81727988/170507178-09e3fe0a-c31b-47db-bc95-36c5db84b1fe.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/170507264-1be8711f-230b-46df-a524-3b636a527a7f.png)